src/legacy/ui/public/styles/_legacy/components/_ui_select.scss

```diff
@@ -1,357 +0,0 @@
-/*!
- * ui-select
- * http://github.com/angular-ui/ui-select
- * Version: 0.19.5 - 2016-10-24T23:13:59.551Z
```

Comment:
I think ML is still using this library? If so, and they're the only ones, this file should live in the ML folder. If not, can we remove the whole dependency? Including the selectors in the `_hacks.scss` file in ML.

Reply:
don't think there's any usage of ui-select left, so we could remove it including hacks, dear @elastic/ml-ui, can you check? many thx

Reply:
Thanks for the ping! We are angular-free now so it should be safe to delete both `_ui_select.scss` and everything related to `ui-select` in `x-pack/plugins/ml/public/application/_hacks.scss`.

Reply:
thx, will do!

Reply:
removed it